index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Landing Page</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <button class="btn"><i class="fas fa-bars fa-2x"></i></button>
+    <div id="container">
+      <div class="heading-primary">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div id="wrapper">
+        <img src="img/10613180113_fdf7bcd316_b.jpg" alt="" />
+        <p>
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <h4 class="text-center">Here's a time line of Dr. Borlaug's life:</h4>
+      <ul>
+        <li>1914 - Born in Cresco, Iowa</li>
+        <li>
+          1933 - Leaves his family's farm to attend the University of Minnesota,
+          thanks to a Depression era program known as the "National Youth
+          Administration"
+        </li>
```

Comment:
- [ ] You've made a polite request [here](https://github.com/DannieMo/Code-Review-/pull/1#pullrequestreview-700823243), but your suggestion lacks a `why`.. 
Please always include an explanation, a "why", for each requested change. For example:
- This is a well-explained request: 
  Why don't you use semantic HTML tags instead of a div, improving the HTML structure and making the page more accessible? 
- This request lacks a "why": 
Why don't you use semantic HTML tags here instead of a div?

- [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:
This is an example of an inline comment. :slightly_smiling_face:
You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

Reply:
Do I make the corrections I reviewed and push again or should I just review and leave the corrections?